build/python/backend/Dockerfile

```diff
@@ -21,6 +21,7 @@ RUN apt-get -qq update && \
     apt-get install --no-install-recommends -qq \
     automake \
     build-essential \
+    cmake \
```

Comment:
Does this fix the build issues? Happy to include as long as it's intentional!

Reply:
I definitely needed it to build on ARM !